src/linear_algebra/finsupp.lean

```diff
@@ -215,11 +216,16 @@ begin
   exact linear_map.is_linear _
 end
 
-/-- `finsupp.sum` as a linear map. -/
-def lsum (f : α → M →ₗ[R] N) : (α →₀ M) →ₗ[R] N :=
-⟨λ d, d.sum (λ i, f i),
-  assume d₁ d₂, by simp [sum_add_index],
-  assume a d, by simp [sum_smul_index', smul_sum]⟩
+/-- Lift a family of linear maps `M →ₗ[R] N` indexed by `x : α` to a linear map map from `α →₀ M` to
+`N` using `finsupp.sum`. We define this as an additive equivalence. For a commutative `R`, this
+equivalence can be upgraded to a linear equivalence. -/
```

Comment:
Can we mention that this is a slightly stronger `finsupp.lift_add_hom` with `→+` replaced by `→ₗ[R]`?